app/controllers/api/v3/collections_controller.rb

```diff
@@ -27,20 +28,20 @@ def index
       end
 
       def show
-        response = etmodel_client.get("/api/v1/transition_paths/#{params.require(:id).to_i}")
+        response = idp_client.get("/api/v1/collections/#{params.require(:id).to_i}")
         render json: response.body
       rescue Faraday::ResourceNotFound
         render_not_found
       end
 
       def create
         UpsertTransitionPath.new(
-          endpoint_path: '/api/v1/transition_paths',
+          endpoint_path: '/api/v1/collections',
           method: :post
         ).call(
           params: params.permit!.to_h,
```

Comment:
```suggestion
          params: params.permit!.to_h.merge(version: Settings.version_tag),
```
Communicate the version on create